full/full_coupler_mod.F90

```diff
@@ -226,6 +228,47 @@ module full_coupler_mod
                          use_hyper_thread, concurrent_ice, slow_ice_with_ocean,    &
                          do_endpoint_chksum, combined_ice_and_ocean
 
+  type coupler_clock_type 
+    integer :: initialization
```

Comment:
These should all be private and we should use getters and setters to get and set the variables.  Please open an issue to address this in the near future.